DataFormats/TrackReco/interface/TrackBase.h

```diff
@@ -114,31 +114,32 @@ namespace reco {
       lowPtQuadStep = 23,
       detachedQuadStep = 24,
       displacedGeneralStep = 25,
-      reservedForUpgrades2 = 26,
-      bTagGhostTracks = 27,
-      beamhalo = 28,
-      gsf = 29,
+      exoRegionalStep = 26,
+      reservedForUpgrades2 = 27,
```

Comment:
when we discussed integration of what's now called `displacedGeneralStep`, the idea for the regional was that it can still reuse `reservedForUpgrades2` and that the naming would be similar, `displacedRegionalStep`

Reply:
OK, I renamed the iteration `exoRegionalStep` → `displacedRegionalStep` and it now occupies the slot previously held by `reservedForUpgrades2`.